core/trino-main/src/main/java/io/trino/operator/scalar/timestamp/ToUnixTime.java

```diff
@@ -1,43 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.operator.scalar.timestamp;
-
-import io.trino.spi.function.LiteralParameters;
-import io.trino.spi.function.ScalarFunction;
-import io.trino.spi.function.SqlType;
-import io.trino.spi.type.LongTimestamp;
-import io.trino.spi.type.StandardTypes;
-
-import static io.trino.type.DateTimes.MICROSECONDS_PER_SECOND;
-import static io.trino.type.DateTimes.PICOSECONDS_PER_SECOND;
-
-@ScalarFunction("to_unixtime")
-public final class ToUnixTime
-{
-    private ToUnixTime() {}
```

Comment:
Update datetime.rst as well

```
.. function:: to_unixtime(timestamp) -> double

    Returns ``timestamp`` as a UNIX timestamp.
```

Reply:
`timestamp` in that context is the name of the argument, not a type

Reply:
isn't that a bit less informative than it could be?
(not to say 'misleading')

Reply:
There's no way to indicate the type in the rst syntax we use, unfortunately. Also, the names of the arguments are not very consistent. We have stuff like this, for instance: `.. function:: date(x) -> date`